docs/cri/config.md

```diff
@@ -161,490 +152,8 @@ See also [the Kubernetes documentation](https://kubernetes.io/docs/concepts/cont
-    # cni config.
-    # If this is set, containerd will generate a cni config file from the
-    # template. Otherwise, containerd will wait for the system admin or cni
-    # daemon to drop the config file into the conf_dir.
-    # See the "CNI Config Template" section for more details.
-    conf_template = ""
-    # ip_pref specifies the strategy to use when selecting the main IP address for a pod.
-    # options include:
-    # * ipv4, "" - (default) select the first ipv4 address
-    # * ipv6 - select the first ipv6 address
-    # * cni - use the order returned by the CNI plugins, returning the first IP address from the results
-    ip_pref = "ipv4"
-    # use_internal_loopback specifies if we use the CNI loopback plugin or internal mechanism to set lo to up
-    use_internal_loopback = false
-
-  # 'plugins."io.containerd.grpc.v1.cri".image_decryption' contains config related
-  # to handling decryption of encrypted container images.
-  [plugins."io.containerd.grpc.v1.cri".image_decryption]
-    # key_model defines the name of the key model used for how the cri obtains
-    # keys used for decryption of encrypted container images.
-    # The [decryption document](https://github.com/containerd/containerd/blob/main/docs/cri/decryption.md)
-    # contains additional information about the key models available.
-    #
-    # Set of available string options: {"", "node"}
-    # Omission of this field defaults to the empty string "", which indicates no key model,
-    # disabling image decryption.
-    #
-    # In order to use the decryption feature, additional configurations must be made.
-    # The [decryption document](https://github.com/containerd/containerd/blob/main/docs/cri/decryption.md)
-    # provides information of how to set up stream processors and the containerd imgcrypt decoder
-    # with the appropriate key models.
-    #
-    # Additional information:
-    # * Stream processors: https://github.com/containerd/containerd/blob/main/docs/stream_processors.md
-    # * Containerd imgcrypt: https://github.com/containerd/imgcrypt
-    key_model = "node"
-
-  # 'plugins."io.containerd.grpc.v1.cri".registry' contains config related to
-  # the registry
-  [plugins."io.containerd.grpc.v1.cri".registry]
-    # config_path specifies a directory to look for the registry hosts configuration.
-    #
-    # The cri plugin will look for and use config_path/host-namespace/hosts.toml
-    #   configs if present OR load certificate files as laid out in the Docker/Moby
-    #   specific layout https://docs.docker.com/engine/security/certificates/
-    #
-    # If config_path is not provided defaults are used.
-    #
-    # *** registry.configs and registry.mirrors that were a part of containerd 1.4
-    # are now DEPRECATED and will only be used if the config_path is not specified.
-    config_path = ""
-```
-
-</p>
-</details>
++ In containerd 2.x see [here](https://github.com/containerd/containerd/blob/main/docs/man/containerd-config1.x.md)
```

Comment:
try to avoid see here 

For example 
`The containerd v2.0 configuration settings for the CRI plugin are located in the [___ section of the containerd config guide](.md#section).`